src/main/resources/templates/db_scripts/create-holdings-table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE holdings
+ALTER COLUMN id TYPE VARCHAR (50);
```

Comment:
Assume we will be adding more columns which define resource here (placeholder for create)?

Reply:
Understand all fields to be stored in jsonb column